examples/both/flake.nix

```
@@ -27,7 +27,8 @@
           # Configurations for Linux (NixOS) machines
           nixosConfigurations = {
             # TODO: Change hostname from "example1" to something else.
-            example1 = self.nixos-flake.lib.mkLinuxSystem "x86_64-linux" {
+            example1 = self.nixos-flake.lib.mkLinuxSystem {
+              nixpkgs.system = "x86_64-linux";
```

The *Platform flags are a bit more consistent and future proof.

```suggestion
              nixpkgs.hostPlatform = "x86_64-linux";
```

Alternatively, if the config was based on `nixos-generate-config`'s output, which would be most of the time, use

```suggestion
              imports = [ ./hardware-configuration.nix ];
```

---

> Alternatively, if the config was based on `nixos-generate-config`'s output, which would be most of the time, use

I think we can mention this in the docs; the example cannot use this, because it must be build-able in CI.

(Personally I use `diskio` which eliminates `harddware-configuration.nix`)